### PR TITLE
Fixed syntax error in async await

### PR DESCRIPTION
**Pull request by 2019UCP1350**
The wrong syntax was creating an error so I fixed it up.

### CODE REVIEW

File: dashboard/v1.1/src/pages/Input/Input.tsx [hunk 1]
```diff
@@ -151,19 +151,17 @@ const Input = (props: InputScreenProps) => {
...
-      } catch (err) {
+        const { data } = await response.json();
+        const routes = data.monitoringRoutes;
+        const duplicateRoutes: string[] = [];
+        routes.forEach(route => {
+          let splitURL: string[] = route.split(/(\s+)/);
+          duplicateRoutes.push(splitURL[2]);
+        });
+      setExistingRoutes(duplicateRoutes);
+      }catch (err) {
```
Comment (aquibbaig) [inline dashboard/v1.1/src/pages/Input/Input.tsx:162]: Let's add a ' ' before catch.

**Review by aquibbaig — CHANGES_REQUESTED** [review]

File: dashboard/v1.1/src/pages/Input/Input.tsx (hunk 1 above)
Comment (2019UCP1350) [reply dashboard/v1.1/src/pages/Input/Input.tsx]: @aquibbaig  
added a  space before catch

**Review by aquibbaig — APPROVED** [review]
LGTM!

**Review by Harkishen-Singh — APPROVED** [review]
Thank you for the fix!

**Pull request merged**